content/applications/hr/payroll.rst

```diff
@@ -491,74 +822,130 @@ the employee's card.
+  :guilabel:`Days`, or :guilabel:`Percent`.
 
-.. image:: payroll/offers.png
+  :guilabel:`Currency` is for a set monetary value, :guilabel:`Days` is for compensation in the form
+  of time off, and :guilabel:`Percent` is for a monetary value awarded that is based upon another
+  metric, such as commissions.
+- :guilabel:`Salary Structure Type`: select which :guilabel:`Salary Structure Type` this rule is
+  nested under, from the drop-down menu.
+- :guilabel:`Value Type`: select how the value is computed, using the drop-down menu. The default
+  options are :guilabel:`Fixed Value`, :guilabel:`Contract Value`, :guilabel:`Payslip Value`,
+  :guilabel:`Sum of Benefits Values`, and :guilabel:`Monthly Total`.
+- :guilabel:`Code`: select the code this rule applies to from the drop-down menu.
+
+.. image:: payroll/resume-net.png
    :align: center
-   :alt: A list of all offers given to a current employee or potential candidate along with the
-         status of the offer.
-
-To create a new offer, click the :guilabel:`New` button. The two required fields are the
-:guilabel:`Contract Template`, and the :guilabel:`Company`. Select the :guilabel:`Contract Template`
-and :guilabel:`Company` from the drop-down menus. Fill in any other details for the offer, such as
-the :guilabel:`Job Title`, :guilabel:`Department`, the :guilabel:`Contract Start Date` and the
-:guilabel:`Offer Validity Date`.
-
-.. toctree::
-   :titlesonly:
-
-   payroll/contracts
-   payroll/payslips
-   payroll/work_entries
-   payroll/reporting
+   :alt: The net wage rule form filled out, with all the information for net pay.
+
+Jobs
+====
+
+Since the *Payroll* application is responsible for paying employees for specific job positions, the
+complete list of job positions can be found in both the *Payroll* and *Recruitment* applications.
+
+.. _payroll/job-positions:
+
+Job positions
+-------------
+
+The job positions listed in the *Payroll* application are identical to the job positions listed in
+the *Recruitment* application. If a new job position is added in the *Recruitment* application, it
+is also visible in the *Payroll* application, and vice versa.
+
+To view the job positions, navigate to :menuselection:`Payroll app --> Configuration --> Jobs: Job
+Positions`.
+
+A list of all the job positions appear, along with the corresponding department, on the
+:guilabel:`Job Position` page.
+
+.. image:: payroll/job-positions.png
+   :align: center
+   :alt: A list of all the job positions and corresponding departments.
+
+To create a new job description, click the :guilabel:`New` button and a job form appears. Enter the
+information on the form for the new position. The information is identical as to the information
+entered when creating a new job position in the *Recruitment* application. Refer to the
+:doc:`../hr/recruitment/new_job` document for more details on how to fill out this form.
```

Comment:
```suggestion
To create a new job description, click the :guilabel:`New` button, and a job form appears.

Enter the information on the form for the new position. The information is identical to the
information entered when creating a new job position in the *Recruitment* application.

Refer to the :doc:`../hr/recruitment/new_job` documentation for more details on how to fill out this
form.
```

Reply:
please make this change during your next revision. I think it may have gotten overlooked last time. Thanks!